srcpkgs/vscode/template

```diff
@@ -1,38 +1,57 @@
 # Template file for 'vscode'
 pkgname=vscode
-version=1.25.1
-revision=2
+version=1.26.1
+revision=1
 hostmakedepends="pkg-config python nodejs-lts yarn"
 makedepends="libX11-devel libxkbfile-devel libsecret-devel"
 depends="GConf libXtst libxkbfile nss"
 short_desc="Microsoft Code for Linux"
-maintainer="Julio Galvan <juliogalvan@protonmail.com>"
+maintainer="shizonic <realtiaz@gmail.com>"
 license="MIT"
 homepage="https://code.visualstudio.com/"
 distfiles="https://github.com/Microsoft/vscode/archive/${version}.tar.gz"
-checksum=af95547a64647d4097c1b45902fc53f45d323f053a7558bfa42e1900621d972c
+checksum=1147b8f665cc8d025d09a5250c284d5a7a2d0d1a6bc5a97eaf3a33864b21339e
+patch_args="-Np1"
+
 # Due to electron
 only_for_archs="i686 x86_64"
 nostrip_files="code-oss"
 
+# Due to this bug:
+# https://github.com/Microsoft/vscode/issues/23946
+_software_rendering="yes"
```

Comment:
i'd remove this and move the comment to above the sed call, also add another line with a short explaination